pkg/resources/external_function_acceptance_test.go

```diff
@@ -482,80 +480,80 @@ resource "snowflake_external_function" "f" {
 func externalFunctionConfigIssue2528(database string, schema string, name string, schema2 string) string {
 	return fmt.Sprintf(`
 resource "snowflake_api_integration" "test_api_int" {
-  name                 = "%[3]s"
-  api_provider         = "aws_api_gateway"
-  api_aws_role_arn     = "arn:aws:iam::000000000001:/role/test"
-  api_allowed_prefixes = ["https://123456.execute-api.us-west-2.amazonaws.com/prod/"]
-  enabled              = true
+ name                 = "%[3]s"
+ api_provider         = "aws_api_gateway"
+ api_aws_role_arn     = "arn:aws:iam::000000000001:/role/test"
+ api_allowed_prefixes = ["https://123456.execute-api.us-west-2.amazonaws.com/prod/"]
+ enabled              = true
 }
 
 resource "snowflake_schema" "s2" {
-  database            = "%[1]s"
-  name                = "%[4]s"
+ database            = "%[1]s"
+ name                = "%[4]s"
 }
 
 resource "snowflake_external_function" "f" {
-  name     = "%[3]s"
-  database = "%[1]s"
-  schema   = "%[2]s"
-  arg {
-    name = "SNS_NOTIF"
-    type = "OBJECT"
-  }
-  return_type = "VARIANT"
-  return_behavior = "VOLATILE"
-  api_integration = snowflake_api_integration.test_api_int.name
-  url_of_proxy_and_resource = "https://123456.execute-api.us-west-2.amazonaws.com/prod/test_func"
+ name     = "%[3]s"
+ database = "%[1]s"
+ schema   = "%[2]s"
+ arg {
+   name = "SNS_NOTIF"
+   type = "OBJECT"
+ }
+ return_type = "VARIANT"
+ return_behavior = "VOLATILE"
+ api_integration = snowflake_api_integration.test_api_int.name
+ url_of_proxy_and_resource = "https://123456.execute-api.us-west-2.amazonaws.com/prod/test_func"
 }
 
 resource "snowflake_external_function" "f2" {
-  depends_on = [snowflake_schema.s2]
-
-  name     = "%[3]s"
-  database = "%[1]s"
-  schema   = "%[4]s"
-  arg {
-    name = "SNS_NOTIF"
-    type = "OBJECT"
-  }
-  return_type = "VARIANT"
-  return_behavior = "VOLATILE"
-  api_integration = snowflake_api_integration.test_api_int.name
-  url_of_proxy_and_resource = "https://123456.execute-api.us-west-2.amazonaws.com/prod/test_func"
+ depends_on = [snowflake_schema.s2]
```

Comment:
was this whitespace change necessary?

Reply:
I'm not sure why that happened, but I'm guessing goland or some go formatter thought it's better that way 🤷